doc/man1/openssl-pkeyutl.pod.in

```diff
@@ -63,20 +63,26 @@ if this option is not specified.
 
 =item B<-rawin>
 
-This indicates that the input data is raw data, which is not hashed by any
-message digest algorithm. The user can specify a digest algorithm by using
-the B<-digest> option. This option can only be used with B<-sign> and
-B<-verify> and must be used with the Ed25519 and Ed448 algorithms.
+This indicates that signature input data is raw data, which for most signature
+algorithms (but not EdDSA) needs to be hashed by some message digest algorithm.
+This option can only be used with B<-sign> and B<-verify>
+and is implied by the Ed25519 and Ed448 algorithms.
+Except with EdDSA,
+the user can specify a digest algorithm by using the B<-digest> option.
+
+The B<-digest> option implies B<-rawin>.
 
 =item B<-digest> I<algorithm>
 
-This specifies the digest algorithm which is used to hash the input data before
+This option can only be used with B<-sign> and B<-verify>.
+It specifies the digest algorithm which is used to hash the input data before
 signing or verifying it with the input key. This option could be omitted if the
 signature algorithm does not require one (for instance, EdDSA). If this option
 is omitted but the signature algorithm requires one, a default value will be
```

Comment:
```suggestion
is omitted (and B<-rawin> is used instead), but the signature algorithm requires one, a default value will be
```

Reply:
Fixing this is part of PR #25831 (also because that one certainly will be backported).